src/pages/Debug/DebugDetails.tsx

```diff
@@ -56,37 +52,28 @@ function DebugDetails({data, onyxKey, isCollection = false, idSelector = () => '
                     setErrors({});
                     const results = Object.entries(draftData).map(([key, value]) => {
                         try {
-                            if (data && typeof data === 'object' && typeof data[key] === 'object') {
-                                // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-                                return [key, typeof value === 'string' ? JSON.parse(value.replaceAll('\n', '')) : value];
+                            if (isObject(data[key])) {
+                                return [key, typeof value === 'string' ? JSON.parse(value.replaceAll('\n', '')) : value] as [string, unknown];
```

Comment:
Why do we need `value.replaceAll('\n', '')`?

Reply:
@fabioh8010 When I apply JSON.stringify(), it adds \n characters which JSON.parse() isn't able to parse, so I added this logic to remove these special characters